specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7056,6 +7113,153 @@
       },
       "description": "The List Extension operation response"
     },
+    "VirtualMachineSoftwarePatchProperties": {
+      "properties": {
+        "patchId": {
```

Comment:
>"patchId" [](start = 8, length = 9)

can you mark all these as readonly for completeness?

Reply:
Sure